app/controllers/concerns/sms.rb

```diff
@@ -2,36 +2,16 @@
 require "twilio-ruby"
 
 # A set of utility methods used in relation to SMS notifications
-# @author Eric Schlange <eric.schlange@northwestern.edu>
 module Sms
-  extend ActiveSupport::Concern
-
-  # Send an SMS message body to a specified receiving participant (leaving
-  # off recipient defaults the value to the current participant)
-  # rubocop:disable Metrics/AbcSize
-  def send_sms(recipient = current_user, message_body)
+  def send_sms(recipient, message_body)
+    return unless Rails.env.staging? || Rails.env.production?
```

Comment:
@ericcf Sorry, that was the wrong link: I'll update the original comment link to this: http://homepages.tig.com.au/~ijoyner/Ian_Joyner/No_return.html

I was reading arguments for both sides, I felt the following post looked at both sides and made a decent argument for SESE:

http://homepages.tig.com.au/~ijoyner/Ian_Joyner/No_return.html